frontend/lib/shared/components/navigation/nav_profile_image.component.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+class NavProfileImage extends StatelessWidget {
+  const NavProfileImage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Hero(
```

Comment:
Same comment as above, I think the Hero animation might not be working (at least on my end). This could be b/c you're using `.replace`(?)